src/main/java/com/learnreactivespring/model/Weather.java

```diff
@@ -0,0 +1,42 @@
+package com.learnreactivespring.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+
+import java.util.List;
+import java.util.Map;
+
+@Document
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+public class Weather {
+
+  @Id
+  String name;
+  List<WeatherModel> weather;
+  MainModel main;
+
+
+  public static class MainModel {
+
+    public Double temp;
+    public Double feels_like;
+    public Double temp_min;
+    public Double temp_max;
+    public Double pressure;
+    public Double humidity;
```

Comment:
- Please use the camelCase for naming the fields.
- These properties should be private.
- Use getter/setter.